src/script/lua_api/l_env.cpp

```diff
@@ -34,10 +34,11 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 #include "content_sao.h"
 #include "treegen.h"
 #include "pathfinder.h"
+#include "face_position_cache.hpp"
 
-#define GET_ENV_PTR ServerEnvironment* env =                                   \
-				dynamic_cast<ServerEnvironment*>(getEnv(L));                   \
-				if (env == NULL) return 0
+#define GET_ENV_PTR ServerEnvironment* env = \
+		dynamic_cast<ServerEnvironment*>(getEnv(L)); \
+	if (env == NULL) return 0
```

Comment:
This should be intended another tab.
Also this isn't related to logging in any way